react-graphql-apollo-tutorial.md

```diff
@@ -55,25 +55,23 @@ In this tutorial, you will learn how to combine React with GraphQL in your appli
 
 {{% chapter_header "Starting with Apollo Boost on the Command Line" "graphql-apollo-boost-client" %}}
 
-Apollo is a whole ecosystem built by developers as a infrastructure for GraphQL applications. You can use it on the client-side for a GraphQL client application, server-side for a GraphQL server application, somewhere in between as a gateway, and also for other GraphQL related tools (e.g. monitoring). At the time of writing this tutorial, Apollo offers the richest and most popular ecosystem around GraphQL in JavaScript. There are other libraries for React applications out there, such as {{% a_blank "Relay" "http://facebook.github.io/relay" %}} and {{% a_blank "Urql" "https://github.com/FormidableLabs/urql" %}}, but they are only for React applications and not as popular as the Apollo Client which has the advantage of being framework agnostic. You don't need to use Apollo necessarily with React. It can be coupled with other view-layer Vue and Angular as well. So everything you will learn in this tutorial about Apollo Client might be transferable to Angular and Vue.
+Apollo is an entire ecosystem built by developers as an infrastructure for GraphQL applications. You can use it on the client-side for a GraphQL client application, server-side for a GraphQL server application, somewhere between as a gateway, or for other GraphQL related tools like monitoring). At the time of writing this tutorial, Apollo offers the richest and most popular ecosystem around GraphQL in JavaScript. There are other libraries for React applications like {{% a_blank "Relay" "http://facebook.github.io/relay" %}} and {{% a_blank "Urql" "https://github.com/FormidableLabs/urql" %}}, but they are just for React applications, and they are not as popular as the Apollo Client. Apollo is framework agnostic, meaning you can use it with other applications than react.  It can be coupled with other view-layers like Vue and Angular as well, so everything you learn in this tutorial is likely transferable to the others.
```

Comment:
"... somewhere between as a gateway, or for other GraphQL related tools like monitoring)". <- remove )

Reply:
"... meaning you can use it with other applications than react." to "... meaning you can use it with other frameworks such as Vue or Angular."